lib/Rest/VstRequest.cpp

```diff
@@ -173,7 +173,10 @@ void VstRequest::parseHeaderInformation() {
     }
 
     // fullUrl should not be necessary for Vst
-    _fullUrl = _requestPath + "?";
+    _fullUrl = _requestPath;
+    if (!_values.empty() || !_arrayValues.empty()) {
+      _fullUrl.push_back('?');
```

Comment:
I think this causes an issue now.
Because there is an unconditional `pop_back()` performed on `_fullUrl` below. And when there are no `_values` and no `_arrayValues`, no `?` and no `&` will be appended to the URL, so the `pop_back()` will remove one legitimate char from the URL.